packages/ckeditor5-image/src/image/imageblock.js

```diff
@@ -0,0 +1,73 @@
+/**
+ * @license Copyright (c) 2003-2020, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md or https://ckeditor.com/legal/ckeditor-oss-license
+ */
+
+/**
+ * @module image/image/imageblock
+ */
+
+import Plugin from '@ckeditor/ckeditor5-core/src/plugin';
+
+import { viewFigureToModel } from './converters';
+
+import { toImageWidget, createImageViewElement, getImageTypeMatcher } from './utils';
+
+/**
+ * The image engine plugin.
+ *
+ * It registers:
+ *
+ * * `<image>` as a block element in the document schema, and allows `alt`, `src` and `srcset` attributes.
+ * * converters for editing and data pipelines.
+ * * `'imageInsert'` command.
+ *
+ * @extends module:core/plugin~Plugin
+ */
+export default class ImageBlock extends Plugin {
+	/**
+	 * @inheritDoc
+	 */
+	static get pluginName() {
+		return 'ImageBlock';
+	}
+
+	/**
+	 * @inheritDoc
+	 */
+	init() {
+		const editor = this.editor;
+		const schema = editor.model.schema;
+		const t = editor.t;
+		const conversion = editor.conversion;
+
+		// Configure schema.
+		schema.register( 'image', {
+			isObject: true,
+			isBlock: true,
+			allowWhere: '$block',
+			allowAttributes: [ 'alt', 'src', 'srcset' ]
```

Comment:
This plugin should also register converters for those attributes.

Reply:
Done.